subtitles/en/parity-magic-trick.vtt

```diff
@@ -11,7 +11,7 @@ Date: 13/02/2017
 
```

Comment:
Maybe say that you edited this file, though the translators didn't say anything either